_posts/2017-11-11-release-0.16.0.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: Release 0.16.0
+---
+
+We have just released MDAnalysis version 0.16.0. This release contains new
+features as well as bug fixes. Highlights are listed below but for more details
+see the [release notes](https://github.com/MDanalysis/mdanalysis/wiki/...).
+
+This is the biggest release we ever had with tons of new features. It includes a
+rewrite of the topology system, the work of our GSoC students Fiona Naughton
+(@fiona-naughton) and John Detlefs (@jdetle), a complete new ensemble analysis
+with [encore]({% post_url 2016-12-17-encore %}) and much more. In total **XX** People
+contributed to this release, with **XX** new contributors and **XX** commits.
+
+# Upgrade
+
+You can upgrade with `pip install --upgrade MDAnalysis` . If you use the conda
+package manager run `conda update -c MDAnalysis mdanalysis`
+
+
+# Noticable Changes
+
+## Rewrite of our internal topology representation
+
+This is the change we are most exited about for this release. It brings
+performance enhancements, makes maintaining the code easier, it's easier to
+extend and allowed us a simplification of the interface. We have previously
+written about the change [here]({% post_url 2016-05-16-release-0.15.0 %})
+and [here](). This new system already allowed us a to implement the [new set
+operations]() for AtomGroups.
+
+But with this change also a lot of deprecated functions have been removed. For
+all of the deprecated functions the new replacements already exists and you will
+get a warning with a suggested change. The easiest way to check if your scripts
+will run without a problem after the update is to include the following on top
+of it.
+
+{% highlight python %}
+import warnings
+warnings.filterwarnings('always', module='MDAnalysis')
+{% endhighlight %}
+
+This will print a warning for every deprecated function you are using together
+with a short code snippet how your code has to be changed. If you do not want to
+upgrade your existing scripts we [posted a guide]() how to use conda and python
+environments to run different versions of MDAnalysis on the same computer.
```

Comment:
Maybe we just link to the wiki page for the release and the changes section there and keep it high-level. The most important thing is to be *aware* that things changed and if there are links to follow then that's good enough. 

Also link to the topology post #41 .

(Good to link to your conda/venv guide #47 !)